CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### Internal
 - [In Progress: 19th May 2020] Migrate `ScheduleAppointmentSheet` to Mobius
 - Fix PR description CI when using markdown syntax
+- Shutdown camera executor when QR scanner view is detached
```

Comment:
Isn't this a fix for the exception?

Reply:
We don't know yet, it's a potential fix. So we want to merge it and continue monitoring to see if crashes are still happening.

Reply:
Can you still move this to the `Fix` section?